core/src/main/java/com/fincity/saas/core/functions/connection/email/SendEmail.java

```diff
@@ -0,0 +1,108 @@
+package com.fincity.saas.core.functions.connection.email;
```

Comment:
move it to com.fincity.saas.core.functions.email, just keeping in sync with the namespace.